app/services/export_service_exporters/response_exporter.rb

```diff
@@ -50,6 +56,25 @@ def replacing_special(text)
       text.gsub(/\s/, '_').gsub(/\W/, '')
     end
 
-    attr_accessor :response_export_service, :responses_to_export, :exports
+    def with_exception_logging
```

Comment:
These 3 methods are the same as the ones in the file above. Maybe a little bit more DRY it?

Reply:
Trouble is, I dont want to do a superclass as it would mean that all exporters will always have the same base which they won't.  
Then, I thought of doing a module but the initialize method would be a problem as you cannot override that in the module - so no way to set exceptions to [] ? - I would not want to include a module that is only partially complete - it either deals with all exception logging in the module or it doesnt - in this case it would be partially complete so I left it.

Im also a strong believer that just because 2 classes share similar code that they should not be dried up - sure if 3, 4 or 5 do then thats ok to do so - we can end up with the wrong inheritance tree if we are not careful that is there just for the sake of DRYing code.

Reply:
Ok fair enough.